frontend/src/components/pages/chat/components/chatList/ChatList.tsx

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { ChatListItem } from '../chatListItem/ChatListItem';
+import ChatSearch from '../chatSearch/ChatSearch';
+import './chatList.scss';
+
+type ChatProps = {
+  picture: {
+    large: string;
+  };
+  name: {
+    first: string;
+    last: string;
+  };
+  text: string;
+};
+
+export const ChatList: React.FC = () => {
+  const [chats, setChats] = useState<Array<string>>([]);
+  useEffect(() => {
+    getChats();
+  }, []);
+
+  const getChats = () => {
+    axios.get('https://randomuser.me/api/?results=20').then(response => {
+      let newChats = response.data.results.map((result: ChatProps) => {
+        return {
+          photo: result.picture.large,
+          name: `${result.name.first} ${result.name.last}`,
+          text: 'Hello world! This is a long message that needs to be truncated.'
+        };
+      });
+      setChats([...chats, ...newChats]);
+    });
+  };
+
+  return (
+    <div className="chat-list">
+      <div className="chat-search">
+        <ChatSearch />
+      </div>
+      <div className="chat-list-container">
+        {chats.map((chat: any) => (
```

Comment:
use type instead of any

Reply:
![image](https://user-images.githubusercontent.com/43353740/96745925-4e6f3100-13cf-11eb-85ef-247603a20a8e.png)
![image](https://user-images.githubusercontent.com/43353740/96745983-60e96a80-13cf-11eb-8bb3-4b8b91077389.png)

Reply:
well what would you expect? 
At line 
`const [chats, setChats] = useState<Array<string>>([]);`
you explicitly tell TS that chats is string array, but during iteration you using any, or ChatInfoProps type... 

const [chats, setChats] = useState<ChatInfoProps[]>([]);
...
{chats.map((chat: ChatInfoProps) => (